packages/react-lowcode/src/codegen/generation/ts/ast.ts

```diff
@@ -1,90 +1,133 @@
 import { astFindSource, SourceLineCol } from "../../../ast";
-import ts, { factory } from "typescript"
-
+import ts, { factory } from "typescript";
 
 export interface Node {
-    getText(includeJsDocComments?: boolean): string;
+  getText(includeJsDocComments?: boolean): string;
 }
 
 export interface Symbol {
-    isAnonymous(): boolean;
-    isAny(): boolean;
-    isArray(): boolean;
-    isBoolean(): boolean;
-    isString(): boolean;
-    isNumber(): boolean;
-    isBooleanLiteral(): boolean;
-    isEnumLiteral(): boolean;
-    isNumberLiteral(): boolean;
-    isStringLiteral(): boolean;
-    isClass(): boolean;
-    isClassOrInterface(): boolean;
-    isEnum(): boolean;
-    isInterface(): boolean;
-    isObject(): boolean;
-    //isTypeParameter(): this is TypeParameter;
-    isTuple(): boolean;
-    isUnion(): boolean;
-    isIntersection(): boolean;
-    isUnionOrIntersection(): boolean;
-    isUnknown(): boolean;
-    isNull(): boolean;
-    isUndefined(): boolean;
+  isAnonymous(): boolean;
+  isAny(): boolean;
+  isArray(): boolean;
+  isBoolean(): boolean;
+  isString(): boolean;
+  isNumber(): boolean;
+  isBooleanLiteral(): boolean;
+  isEnumLiteral(): boolean;
+  isNumberLiteral(): boolean;
+  isStringLiteral(): boolean;
+  isClass(): boolean;
+  isClassOrInterface(): boolean;
+  isEnum(): boolean;
+  isInterface(): boolean;
+  isObject(): boolean;
+  //isTypeParameter(): this is TypeParameter;
+  isTuple(): boolean;
+  isUnion(): boolean;
+  isIntersection(): boolean;
+  isUnionOrIntersection(): boolean;
+  isUnknown(): boolean;
+  isNull(): boolean;
+  isUndefined(): boolean;
 }
 
-export function findWidgetDefinition(code: string, position: SourceLineCol){
-    const found = astFindSource(code, position)
-    
-    if (found) {
-        let before = code.substring(0, found.end)
-        let startOfWidget = before.lastIndexOf('export')
+export function findWidgetDefinition(code: string, position: SourceLineCol) {
+  const found = astFindSource(code, position);
+
+  if (found) {
+    let before = code.substring(0, found.end);
+    let startOfWidget = before.lastIndexOf("export");
```

Comment:
What if there are multiple exports and I also decide to export some constant at the beginning of file... ?

Reply:
This is obsolete function which is not used. I removed it.